drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -431,18 +434,53 @@ function DragAndDropTemplates(configuration) {
                             {
                                 innerHTML: gettext("Close")
                             }
-                        ),
+                         ),
                         h(
                             'span.icon.fa.fa-times-circle',
                             {
                                 attributes: {
                                     'aria-hidden': true
                                 }
                             }
+                         )
+                    ]
+                 ),
+                h(
+                    ctx.last_action_correct ? 'div.popup-header-icon' : 'div.popup-header-icon.popup-header-icon-incorrect',
+                    {},
+                    [
+                        h(
+                            ctx.last_action_correct ? 'span.icon.fa.fa-check-circle' : 'span.icon.fa.fa-exclamation-circle',
+                            {
+                                attributes: {
+                                    'aria-hidden': true
+                                }
+                            }
                         )
                     ]
+                  ),
+                h(
+                    'div.popup-header-text',
+                    {
+                        innerHTML: gettext(ctx.last_action_correct ? "Correct" : "Incorrect")
```

Comment:
This needs to be rewritten as:

```
innerHTML: ctx.last_action_correct ? gettext("Correct") : gettext("Incorrect")
```

Otherwise gettext will not be able to parse the strings.